.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: ruby
 
 rvm:
-  - 2.3
+  - 2.4
```

Comment:
Here you needed to use at least version 2.4 for ruby, even tho I'd recommend you using more recent versions to avoid any error related to versions again.